src/Style/Display/AccordionStyle.ts

```diff
@@ -1,59 +1,106 @@
-import styled from 'styled-components';
-import { SecondaryColor } from '../Colors';
+import styled, { css } from 'styled-components';
 
-export const AccordionContainer = styled.div`
+import { SecondaryColor, Greyscale } from '../Colors';
+import { IconPosition } from '../../Display/Accordion/Accordion';
+
+const iconSize = 15;
+const iconMargin = 30;
+const offsetIcon = iconSize + iconMargin;
+const transitionTiming = '0.24s';
+
+export const PanelWrapper = styled.div`
   position: relative;
+  border-top: solid 1px ${Greyscale.grey};
+  outline: none;
 `;
 
-export const AccordionContentWrapper = styled.div`
+export const Container = styled.div`
   position: relative;
-  display: grid;
-  grid-template-columns: max-content 1fr;
-  grid-gap: 0.6em;
-  font-size: 1em;
-  line-height: 1.5;
-  padding: 1.5em 0;
+
+  ${PanelWrapper}:first-child {
+    border-top: none;
+  }
+`;
+
+export const IconLabelWrapper = styled.div<IconLabelWrapper>`
+  position: relative;
+  display: flex;
+  justify-content: ${({ position }) =>
+    position === 'left' ? 'flex-start' : 'space-between'};
+  margin-top: 30px;
```

Comment:
I think we should use `padding-top` to prevent margin-collapsing.

The height of < Accordion /> doens't include the margin-top of first < Accordion.Panel />
![image](https://user-images.githubusercontent.com/13249748/72313704-25df3b80-36c7-11ea-83fc-612d7ab6fa1b.png)

Reply:
Oh actually this is intentional because according to the design, the hover size should not include the margin-top :pray: https://app.zeplin.io/project/5ba343bf0c399c33db483653/screen/5ba358620049d012fcb8b234